__snapshots__/layout/_template.spec.js.snap

```diff
@@ -1615,10 +1449,10 @@ exports[`base page template matches the full configuration snapshot 1`] = `
                     
                             
     <a href="#0" class="ons-external-link" target="_blank" rel="noopener">
-        <span class="ons-external-link__text">UK Statistics Authority</span><span class="ons-external-link__icon">&nbsp;<svg class="ons-icon" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
```

Comment:
I think this `aria-hidden="true"` `fill="currentColor"` change needs to be reverted. Just edit the icon in the icon component